frontend/layouts/MainAppLayout/NavBar/Navbar.jsx

```diff
@@ -126,7 +131,7 @@ const NavBar = () => {
         return isNotHomePage ? false : homeRegex.test(pathname);
 
       // TODO: Once Discovery Feature is ready, uncomment below statement.
-      // if (id === 'discovery') return discoveryRegex.test(pathname);
+      if (id === 'discovery') return discoveryRegex.test(pathname);
```

Comment:
Let's not directly use strings like above, store these literals in a constant file and reference them here

`${ROUTES_IDS.DISCOVERY}`